default.htm

```diff
@@ -187,12 +187,12 @@ <h2>Read Domesday</h2>
                 </a>
             </div>
             <div class="holder">
-                <a href="https://www.lincolncastle.com/domesday" target="_blank">
-                    <h2>See Domesday in Lincoln</h2>
+                <a href="http://blog.nationalarchives.gov.uk/blog/domesday-book-brief-material-history/" target="_blank">
+                    <h2>Domesday Book</h2>
                         <div class="image-wrapper">
-                            <img src="img/domesday-lincolncastle-1200-x-630.jpg" alt="lincoln-castle-event">
+                            <img src="https://blog.nationalarchives.gov.uk/wp-content/uploads/2017/05/d3.jpg" alt="domesday-book">
```

Comment:
This image is quite big - can we use a smaller, local version to save loading the larger file?